dashboard/app/views/layouts/_parent_email_banner.haml

```diff
@@ -0,0 +1,32 @@
+-# Only show when we're not showing the cookie banner,
+-# the current user is signed in, a student, doesn't have a parent email, and hasn't previously dismissed the banner
+-# and the page isn't an Applab or Gamelab level.
+
+:ruby
+  show_banner =
+    !(request.gdpr? || (rack_env == :test && request.params['show_cookie_banner_on_test'].present?)) &&
```

Comment:
Why is the banner hidden for GDPR countries?

Reply:
Because we show the cookie banner for those users

Reply:
My bad. I thought the cookie banner might be like this one: it doesn't show up if the user has accepted it already. If it wasn't isn't shown, then we could show this one...

I've never seen the cookie banner and don't know the business logic.